Exercises-4-OOP/src/inheritanceperson/Person.java

```diff
@@ -0,0 +1,37 @@
+package inheritanceperson;
+
+public class Person {
+	String name;
```

Comment:
Done for Student class and Staff class.
Can't do private for person class because person class is parent class, if I made variable private then other class can not do inheritance from variable "name", "address"